apps/books/templates/footer.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+</head>
+<body>
+<hr>
+<div>
+    <h1>Footer</h1>
+</div>
+</body>
+</html>
```

Comment:
We don't need to write complete html but only header thing as we know it will be embeded in a parent html file where these tags will already be added

Reply:
Also move this file to some generic/common place